src/components/Keycodes.vue

```diff
@@ -116,6 +125,19 @@ export default {
       this.clearTimeout = window.setTimeout(() => {
         store.commit('app/setMessage', '');
       }, 3000);
+    },
+    filterClass(key) {
+      if (this.searchFilter === '' || isUndefined(key.code)) {
```

Comment:
Why not `if (!this.searchFilter` ?

Reply:
```
s=''
""
!s
true
```

Reply:
I think this is counter intuitive and harder to read. It would only be slightly better if it were !!s